File: Beginner/Chef and Serves (CHSERVE)/serves.cpp
#include <iostream>
using namespace std;

int main() {
	long long t;
	cin>>t;
	while(t--) {
	    long long p1,p2,k;
	    cin>>p1>>p2>>k;
	    long long total=p1+p2;
	    long long x=total/k;
	    if(x%2==0)
	    cout<<"CHEF"<<endl;
	    else
	    cout<<"COOK"<<endl;
	}
	return 0;
}

File: Beginner/Chef and Serves (CHSERVE)/serves.py
t = int(input())

while(t):
    t-=1
    input_text = [int(i) for i in input().split(' ')]
    p1, p2, k = input_text
    total = p1 + p2
    x = int(total/k)
    if x%2==0:
        print("CHEF")
    else:
        print("COOK")
